sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -51,15 +58,19 @@ export class LogsQueryClient {
    * @param options - Options for the LogsClient.
    */
   constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions) {
-    const authPolicy = bearerTokenAuthenticationPolicy(tokenCredential, defaultMonitorScope);
+    const authPolicy = bearerTokenAuthenticationPolicy(
+      tokenCredential,
+      options?.scopes ?? defaultMonitorScope
+    );
 
     // This client defaults to using 'https://api.loganalytics.io/v1' as the
     // host.
     const serviceClientOptions = createPipelineFromOptions(options || {}, authPolicy);
 
     this._logAnalytics = new AzureLogAnalytics({
```

Comment:
.NET made the endpoint a required parameter

Reply:
Why is it required? There is a clear default (that's probably right for most people), right?

Reply:
It would be the first .NET client not to have an explicit endpoint so I kept it consistent with existing ones.

Reply:
But I'm open to making it optional.

Reply:
Resolving for now, we'll figure it out. 

With it as it is we're inline with Java and Python.